print_nanny_webapp/alerts/models.py

```diff
@@ -82,6 +93,42 @@ class AlertSettings(PolymorphicModel):
     )
 
 
+class MethodSettings(PolymorphicModel):
```

Comment:
I can see where you're going with this, but I don't think `enabled` belongs here. An alert method is enabled when its included in the `AlertSettings.alert_methods` array. Happy to talk about restructuring these settings data models as we discover new requirements, but let's do that in a separate PR.

Reply:
so at this point, DiscordSettings should be just a top-level model on its own, right?

Reply:
Yeah. I think your intuition that this will eventually be a Polymorphic model is right though. Let's circle back after 2-3 method settings and figure out which bits they have in common?